crates/bevy_render/src/texture/image.rs

```diff
@@ -831,6 +831,8 @@ impl RenderAsset for Image {
         let texture = render_device.create_texture_with_data(
             render_queue,
             &self.texture_descriptor,
+            // TODO: Is this correct? Do we need to use `MipMajor` if it's a ktx2 file?
```

Comment:
Yes we do, according to the docs.

Reply:
`::default()` is equivalent to what we were doing before, and I can't visually see a difference with my ktx2 bistro if I change it to mipmajor, but I'm bad at spotting texture bugs.